packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -4,7 +4,11 @@
 
 import 'package:flutter/material.dart';
 
-import '../../shared/screen.dart';
+import '../../../devtools_app.dart';
```

Comment:
do not import devtools_app.dart directly inside of `lib/`. This is fine to import in tests, but inside lib we should import the libraries we need directly instead of importing this large file that exports many files.